RELEASING.md

```diff
@@ -43,13 +43,20 @@ Within this repository, this is achieved by the following mechanisms:
 
 - Individual csproj files do not specify a version
 - The [Directory.Build.props](src/Directory.Build.props) file has a `<Version>` element
-  specifying the version of all packages
+  specifying the version of all packages which don't need a separate major
 
 A single GitHub release (and tag) will be created for each beta release, to cover all packages.
 
 - Example tag name: "CloudNative.CloudEvents.All-2.0.0"
 - Example release title: "All packages version 2.0.0"
 
+### Exception: new major versions
+
+Sometimes, we need a new major version of a "satellite" package, typically
```

Comment:
```suggestion
Usually, all packages shipped from this repo share the same major version. Sometimes, we need a new major version of a "satellite" package, typically
```

Reply:
Hmm... I think I'll reword again. (Because it's a "usually in the sense of most packages" rather than a "usually in the sense of most releases".) Thanks for pointing out that it needs tweaking :)

Reply:
I've reworded this whole paragraph and added the specific MQTT example (in the past tense, to avoid having to update it again after the release I expect to do as soon as this is in).

Reply:
Rewording looks good! I mostly wanted to make sure we documented that a different major version was the exception, not the norm, with our current versioning strategy. I think your latest delta conveys that sufficiently.